packages/frontend/src/components/RichTextEditor/index.tsx

```diff
@@ -171,6 +171,9 @@ const Editor = ({
             className="editor__content"
             editor={editor}
             onFocus={() => setShowVarSuggestions(true)}
+            style={{
+              minHeight: isRich ? '9rem' : '2.625rem', // to simulate textarea and coordinate with dropdown input size
```

Comment:
`9rem` might be confusing because when the rich text editor is empty, it's not clickable after the 1st line.

What about just setting these in the css:
```
.ProseMirror {
  padding: 0.5rem 14px;

  // To match DS input sizes
  min-height: 2.625rem;
```

Reply:
stacey wants it to look like a textarea if `isRich` is true, so i did some modifications and now it is clickable!